modules/rsyslog/templates/appslog.conf.erb

```diff
@@ -1,9 +1,9 @@
 #### SINGLE TASK CONF ####
 
 input(type="imfile" 
-    File="<%= @app['log_name'] %>" 
-    Tag="<%= @app['log_tag'] %>" 
-    Severity="<%= @app['severity'] %>"
+    File="<%= @log_name %>" 
+    Tag="<%= @log_tag %>" 
+    Severity="<%= @severity %>"
     Facility="local7"
```

Comment:
I'd also add "facility" as parameter in order to  have more control on logs on remote server but it's not necessary.